alvr/client_openxr/Cargo.toml

```diff
@@ -25,18 +25,18 @@ libc = "0.2"
 ndk-context = "0.1"
 
 [package.metadata.android]
-package = "alvr.client"
+package = "org.meumeu.wivrn2"
```

Comment:
This is pretty funny though

Reply:
Soon

Reply:
this is becoming like a meme for me, constantly forgetting to change it